src/client/components/eventInfo/EventInfo.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './EventInfo.css';
+
+export interface Props {
+  eventName: string;
+  eventDate: string;
+}
+
+export const EventInfo = (props: Props) => {
+  return (
+    <div className="eventInfo">
+      <h1>{props.eventName}</h1>
+      <h2>{props.eventDate}</h2>
```

Comment:
nene, ved to je date from a date to ... cize nemoze byt len tak ze eventdate.

Reply:
ok, myslel som ze vypocet bude nad tym a vlozi sa ako string